### PR TITLE
Test on multiple render api's

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: skiko/src/jvmTest/kotlin/org/jetbrains/skiko/util/UiTest.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import org.junit.Assume.assumeTrue
+import java.awt.GraphicsEnvironment
+
+fun uiTest(block: suspend CoroutineScope.() -> Unit) {
+    assumeFalse(GraphicsEnvironment.isHeadless())
+    assumeTrue(System.getProperty("skiko.test.ui.enabled", "false") == "true")
+
+    runBlocking(Dispatchers.Swing) {
+        if (System.getProperty("skiko.test.ui.allRenderApi", "true") == "true") {
+            SkikoProperties.fallbackRenderApiQueue.forEach {
```
[the remark this answers is not in the document]
Comment (igordmn) [reply skiko/src/jvmTest/kotlin/org/jetbrains/skiko/util/UiTest.kt]: Now we don't override system properties, we pass UiTestScope, which set renderApi for us

File: skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt [hunk 2]
```diff
@@ -3,41 +3,41 @@ package org.jetbrains.skiko
...
             parseRenderApi(property)
         }
     }
 
-    private fun parseRenderApi(text: String?): GraphicsApi {
+    internal fun parseRenderApi(text: String?): GraphicsApi {
         when(text) {
-            "SOFTWARE" -> return GraphicsApi.SOFTWARE
-            "DIRECT_SOFTWARE" -> return GraphicsApi.DIRECT_SOFTWARE
+            "AWT_SOFTWARE" -> return GraphicsApi.AWT_SOFTWARE
```
Comment (olonho) [inline skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt:39]: Didn't we agree that it will be FAST_SOFTWARE and COMPAT_SOFTWARE?

**Review by olonho — APPROVED** [review]
Looks good, modulo (probably unrelated) change related to soft renderers.

File: skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt (hunk 2 above)
Comment (igordmn) [reply skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt]: Thanks for noticing! Forget to revert in this PR. Done

**Pull request merged**